packages/garbo-lib/package.json

```diff
@@ -7,7 +7,14 @@
   "scripts": {
     "build": "echo \"Nothing to do\"",
     "format": "eslint src --fix && prettier --write .",
-    "lint": "eslint src && prettier --check ."
+    "lint": "eslint src && prettier --check .",
+    "pre-commit": "lint-staged"
```

Comment:
Should probably still have madge to check for circular dependencies (also in CI?)

Reply:
Yeah sure. I'll do it later though.